crates/communication/Cargo.toml

```diff
@@ -1,26 +1,27 @@
 [package]
-name = "communication"
-version = "0.1.0"
 edition = "2021"
-license = "GPL-3.0-only"
 homepage = "https://github.com/hulks/hulk"
+license = "GPL-3.0-only"
+name = "communication"
+version = "0.1.0"
 
 [dependencies]
-awaitgroup = { workspace = true }
-bincode = { workspace = true }
-byteorder = { workspace = true }
-color-eyre = { workspace = true }
-framework = { workspace = true, optional = true}
-futures-util = { workspace = true }
-log = { workspace = true }
-serde = { workspace = true }
-serde_json = { workspace = true }
-serialize_hierarchy = { workspace = true }
-thiserror = { workspace = true }
-tokio = { workspace = true }
-tokio-tungstenite = { workspace = true }
-tokio-util = { workspace = true }
-uuid = { workspace = true }
+awaitgroup = {workspace = true}
+bincode = {workspace = true}
+byteorder = {workspace = true}
+color-eyre = {workspace = true}
+framework = {workspace = true, optional = true}
+futures-util = {workspace = true}
+log = {workspace = true}
+parameters = {workspace = true}
+serde = {workspace = true}
+serde_json = {workspace = true}
+serialize_hierarchy = {workspace = true}
+thiserror = {workspace = true}
+tokio = {workspace = true}
+tokio-tungstenite = {workspace = true}
+tokio-util = {workspace = true}
+uuid = {workspace = true}
```

Comment:
@julianschuler Do you have an opinion on TOML formatting?

Reply:
This also surprised me, either an extension I got on vscode messed up or something else is going on.
Would be nice if we can formalize this.

Reply:
Alphabetically and with spaces between the curly braces and items. This change should also be reverted.

Reply:
Fixed, Once I autoformatted other cargo files, I found places where our manual editing didn't order all properly. I'll open a PR later to clean this.